locale/en/manageiq.po

```diff
@@ -4014,6 +4014,12 @@ msgstr ""
 msgid "Already has an ISO datastore"
 msgstr ""
 
+msgid ""
+"Alternative URL path for the XCCDF file, where a com.redhat.rhsa-RHEL7.ds.xml."
+"bz2 file is expected. example: http://my_file_server.example.org:3333/xccdf_fi"
```

Comment:
This reads weird because the really long filename is used as an adjective to describe the file.  Also, missing `For`.  How about this?

```
"Alternative URL path for the XCCDF file, where a file named: com.redhat.rhsa-RHEL7.ds.xml."
"bz2 is expected. For example: http://my_file_server.example.org:3333/xccdf_fi"
```

Reply:
Looking at the changes in https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/539/files, example is following the convention elsewhere in the other help texts and I guess this reads hard here but in the context of the message displayed to users, it's not so bad.  I'll leave it since I don't have a much better way to phrase it.